gridscale/provider.go

```diff
@@ -77,6 +77,7 @@ func Provider() terraform.ResourceProvider {
 			"gridscale_snapshotschedule":               resourceGridscaleStorageSnapshotSchedule(),
 			"gridscale_backupschedule":                 resourceGridscaleStorageBackupSchedule(),
 			"gridscale_paas":                           resourceGridscalePaaS(),
+			"gridscale_k8s":                            resourceGridscaleK8s(),
```

Comment:
Do we want this to be short and sweet "gridscale_k8s" or "gridscale_kubernetes_cluster" as in https://github.com/gridscale/terraform-provider-gridscale/issues/120? 

@itakouna

Reply:
@itakouna please take a look. Thanks!